src/components/button/utils.tsx

```diff
@@ -6,8 +6,10 @@ import { LoadingIcon } from '../loading';
 export type ButtonModes =
   | 'interactive_0'
   | 'interactive_0--bordered'
+  | 'interactive_0--bordered_alternative'
```

Comment:
I think we can use a hyphen instead of an underscore to separate "bordered" and "alternative" → 'interactive_0--bordered-alternative'

Reply:
Okey, I´ll fix👍